charts/yourls/templates/NOTES.txt

```diff
@@ -89,19 +89,15 @@ You can access Apache Prometheus metrics following the steps below:
 {{- include "common.warnings.rollingTag" .Values.image }}
 {{- include "common.warnings.rollingTag" .Values.metrics.image }}
 {{- include "common.warnings.rollingTag" .Values.volumePermissions.image }}
-{{- $passwordValidationErrors := list -}}
 {{- if not .Values.yourls.existingSecret -}}
-    {{- $secretName := include "yourls.secretName" . -}}
-    {{- $requiredYOURLSPassword := dict "valueKey" "yourls.password" "secret" $secretName "field" "password" "context" $ -}}
-    {{- $requiredYOURLSPasswordError := include "common.validations.values.single.empty" $requiredYOURLSPassword -}}
-    {{- $passwordValidationErrors = append $passwordValidationErrors $requiredYOURLSPasswordError -}}
-{{- end }}
-{{- if .Values.mysql.enabled }}
-    {{- $mysqlSecretName := include "yourls.databaseSecretName" . -}}
-    {{- $mysqlPasswordValidationErrors := "OK" -}}
```

Comment:
The issue is coming from here, the rest of the changes aren't needed.

Reply:
Actually, I tested with and without each block and this block did not cause any issue with the validation.